src/database.rs

```diff
@@ -188,6 +188,39 @@ impl Database {
         }
     }
 
+    /** Copies a database file to a new location, and assigns it a new internal UUID to distinguish
+    it from the original database when replicating. */
+    pub fn copy_file<P: AsRef<Path>>(
+        current_db_full_path: P,
+        new_db_name: &str,
+        config: Option<DatabaseConfiguration>,
+    ) -> Result<()> {
+        unsafe {
+            let config = config.map(|cfg| {
+                let mut c_config: CBLDatabaseConfiguration = CBLDatabaseConfiguration_Default();
+                c_config.directory = from_str(cfg.directory.to_str().unwrap()).get_ref();
```

Comment:
No problem with unwrap?

Reply:
Both unwraps removed, thanks for pointing them out :-)